addons/api/.eslintrc.js

```diff
@@ -2,13 +2,17 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
       legacyDecorators: true,
     },
+    requireConfigFile: false,
```

Comment:
This is the approach to NOT have to use a specific babel config file. Because we are just adding 1 plugin I think is not enough to justify such file. But happy to discuss if you think it deserves to have a babel config file aka `.babelrc`